### PR TITLE
rooms are now ordered

### PR DESCRIPTION
**Pull request by jennie-quinton**
Rooms are now ordered so latest rooms is at the top (ignoring saving update at date)

### CODE REVIEW

File: thread/src/components/Threads.js [hunk 1]
```diff
@@ -63,10 +63,25 @@ class Threads extends Component {
         );
     }
 
+    orderRooms(rooms) {
+        rooms.sort((a,b) => {
+            var date1 = new Date(a.updatedAt);
```
Comment (jordanrios94) [inline thread/src/components/Threads.js]: I prefer block scoped `const`

File: thread/src/components/Threads.js [hunk 2]
```diff
@@ -63,10 +63,25 @@ class Threads extends Component {
...
+        });
+
+        return rooms;
+    }
+
     renderThreadList() {
         const { rooms } = this.props;
-
-        if (rooms.length === 0) {
+        orderedRooms = this.orderRooms(rooms);
```
Comment (jordanrios94) [inline thread/src/components/Threads.js]: Variable declaration is missing

File: thread/src/components/Threads.js (hunk 1 above)
Comment (jennie-quinton) [reply thread/src/components/Threads.js]: oops, sorry forgot my es6 for a second

File: thread/src/components/Threads.js (hunk 2 above)
Comment (jennie-quinton) [reply thread/src/components/Threads.js]: 🤦‍♀️

File: thread/src/components/Threads.js [hunk 3]
```diff
@@ -63,10 +63,25 @@ class Threads extends Component {
         );
     }
 
+    orderRooms(rooms) {
```
[the remark this answers is not in the document]
Comment (jennie-quinton) [reply thread/src/components/Threads.js]: good point, i took it out into a seperate function incase we needed to order it in a more complex way, turns out we dont need to :P

**Pull request merged**